test/kubernetes/e2e/features/client_tls/suite.go

```diff
@@ -36,139 +36,91 @@ func NewTestingSuite(ctx context.Context, testInst *e2e.TestInstallation) suite.
 }
 
 func (s *clientTlsTestingSuite) SetupSuite() {
-	err := s.testInstallation.Actions.Kubectl().ApplyFile(s.ctx, testdefaults.NginxPodManifest)
+	err := s.testInstallation.Actions.Kubectl().Apply(s.ctx, testdefaults.NginxPodYaml)
```

Comment:
While we're in here, does it make sense to update this test to use the BaseTestingSuite?